.github/workflows/graph-pixi-tests.yml

```diff
@@ -0,0 +1,127 @@
+name: Graph Pixi Tests
+
+on:
+  push:
+    paths:
+      - 'cypress/e2e/pixi-interaction/**'
+      - '.github/workflows/graph-pixi-tests.yml'
+
+jobs:
+  lint:
+    name: Run Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: snow-actions/sparse-checkout@v1.2.0
+        with:
+          patterns: v3
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: 'npm'
+          cache-dependency-path: v3/package-lock.json
+      - name: Install Dependencies
+        working-directory: v3
+        run: npm ci
+      - name: Restore ESLint Cache
+        id: restore-eslint-cache
+        uses: actions/cache/restore@v4
+        with:
+          path: v3/.cache/eslint/
+          key: eslint-${{ hashFiles('v3/package-lock.json', 'v3/.eslintrc.build.cjs', 'v3/.eslintrc.cjs') }}
+      - name: Run Lint Check
+        working-directory: v3
+        run: npm run lint:build
+      - name: Save ESLint Cache
+        uses: actions/cache/save@v4
+        if: github.ref_name == 'main' || steps.restore-eslint-cache.outputs.cache-hit != 'true'
+        with:
+          path: v3/.cache/eslint/
+          key: ${{ steps.restore-eslint-cache.outputs.cache-primary-key }}
+
+  jest:
```

Comment:
I don't think it is necessary to run the jest tests here. They should already be run by the main workflow.  But perhaps you added them just make sure they were working here since they are failing in the main workflow.